.changeset/light-socks-hear.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-azure-sites-backend': minor
+'@backstage/plugin-azure-sites-common': minor
+'@backstage/plugin-azure-sites': minor
+---
+
+Implemented Permission framework to protect start and stop `Azure sites`
+Fixed issue #16035
```

Comment:
```suggestion
Implemented Permission framework to protect start and stop `Azure sites`.
```

We don't typically reference issues in changesets, but instead describe the fix itself.